packages/app/config/webpack.common.js

```diff
@@ -315,6 +315,7 @@ module.exports = {
   resolve: {
     mainFields: ['browser', 'module', 'jsnext:main', 'main'],
     modules: [
+      path.resolve(__dirname, '../node_modules'),
```

Comment:
Is this one needed as well?

Reply:
Yes, otherwise webpack resolves the below `node_modules` to the hoisted root `node_modules`, and not `app/node_modules`.

Reply:
Hmm, that's weird, oh well!